Intact.BuinessLogic/Services/ProfilesService.cs

```diff
@@ -44,9 +53,10 @@ public async Task<IEnumerable<Profile>> GetAsync(Guid userId, CancellationToken
                 Status = "",
             };
             
-            await _appDbContext.Profiles.AddAsync(profile, cancellationToken);
+            await _redisCache.AddAsync(cacheSet, userId.ToString(), profileDao);
+            await _appDbContext.Profiles.AddAsync(profileDao, cancellationToken);
```

Comment:
probably DB call should be first, and cache is second, in case if db call fails , we do not store cached profile